scripts/eval/yamls/long_context_tasks.yaml

```diff
@@ -0,0 +1,388 @@
+icl_tasks:
+-
+  label: kv_pairs_beginning_2k
+  dataset_uri: hf://maxisawesome/long_context_eval
```

Comment:
we should move these datasets into the mosaicml org, or find a different way to distribute them then via your personal HF account